tests/unit/models/reddit/test_live.py

```diff
@@ -91,6 +91,12 @@ def test_str(self):
         assert str(thread) == 'dummy'
 
 
+class TestLiveThreadContribution(UnitTest):
+    def test_update__no_args(self):
+        thread = LiveThread(self.reddit, 'xyu8kmjvfrww')
+        assert thread.contrib.update() is None
```

Comment:
I think update always returns `None` even if requests are made. By just calling `thead.contrib.update()` however, this is a useful test because it asserts that no requests are made.